overlord/snapstate/snapstate_test.go

```diff
@@ -3474,7 +3467,6 @@ func (s *snapmgrTestSuite) TestAbortCausesNoErrReport(c *C) {
 	s.settle(c)
 
 	c.Check(chg.Status(), Equals, state.UndoneStatus)
-	c.Assert(errReported, Equals, 0)
 }
 
 func (s *snapmgrTestSuite) TestErrreportDisable(c *C) {
```

Comment:
should we remove the full tests, they seem to be really specifically about error reports

Reply:
there were 2 tests to remove I think?